superset/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 - Python package pydruid added ([#110]).
+- Call `superset init` before starting Superset ([#114](https://github.com/stackabletech/docker-images/pull/114))
```

Comment:
This is not part of `superset-stackable2.1.0` and must be moved to a new version. The other entries do not reference the URL inline.